sources/packages/backend/apps/api/src/route-controllers/student-account-applications/models/student-account-application.dto.ts

```diff
@@ -1,8 +1,12 @@
+import { JsonMaxSize } from "../../../utilities/class-validation";
 import { Allow, IsNotEmptyObject } from "class-validator";
 import { CreateStudentAPIInDTO } from "../../../route-controllers/student/models/student.dto";
 
+const JSON_10KB = 10240;
```

Comment:
Must be a constant shared by multiple DTOs.  We can declare them in a constant file created in `route-controllers/utils`

Reply:
We also have the `src/constants`.

Reply:
ok. Done. Thanks!